coding/sensorML-v101/src/main/java/org/n52/sos/encode/SensorMLEncoderv101.java

```diff
@@ -852,9 +857,12 @@ private MethodPropertyType createMethod(final ProcessMethod method) throws Coded
         final List<Characteristics> characteristicsList =
                 Lists.newArrayListWithExpectedSize(smlCharacteristics.size());
         for (final SmlCharacteristics sosSMLCharacteristics : smlCharacteristics) {
+        	final Characteristics xbCharacteristics =
+                    Characteristics.Factory.newInstance(XmlOptionsHelper.getInstance().getXmlOptions());
```

Comment:
to make the transition to 5.x easier, you should move these instance calls to member methods...